src/renderer/components/SetupTab/index.jsx

```diff
@@ -354,12 +379,27 @@ export default class SetupTab extends React.Component {
               argsEnabled={argsEnabled}
               argsDropdownOptions={argsDropdownOptions}
               argsOrder={uiSpec.order}
-              pyModuleName={pyModuleName}
               updateArgValues={this.updateArgValues}
-              batchUpdateArgs={this.batchUpdateArgs}
+              loadParametersFromFile={this.loadParametersFromFile}
             />
           </Row>
           <Portal elId={sidebarSetupElementId}>
+            <OverlayTrigger
+              placement="right"
+              delay={{ show: 250, hide: 400 }}
+              overlay={(
+                <Tooltip>
+                  Browse to a datastack (.json) or invest logfile (.txt)
+                </Tooltip>
+              )}
+            >
+              <Button
+                onClick={this.browseForDatastack}
+                variant="link"
+              >
+                Load parameters from file

```

Comment:
When I made the app window narrower, there's a point where this text wraps to a second line and becomes centered, making it appear indented from the other sidebar options. It's not very long so maybe we could force the sidebar to always stay wide enough for this text?
![image](https://user-images.githubusercontent.com/43770515/125700664-c6084958-fcdc-419b-8bc3-2a2a3dc18567.png)

Reply:
Thanks! I believe I fixed this with `white-space: nowrap` on the button text, and `min-width: fit-content` on the sidebar column.

Reply:
Thanks, it works smoothly now